wormchain/proto/wormhole/tx.proto

```diff
@@ -8,9 +8,12 @@ option go_package = "github.com/wormhole-foundation/wormchain/x/wormhole/types";
 
 // Msg defines the Msg service.
 service Msg {
-      rpc ExecuteGovernanceVAA(MsgExecuteGovernanceVAA) returns (MsgExecuteGovernanceVAAResponse);
+  rpc ExecuteGovernanceVAA(MsgExecuteGovernanceVAA) returns (MsgExecuteGovernanceVAAResponse);
   rpc RegisterAccountAsGuardian(MsgRegisterAccountAsGuardian) returns (MsgRegisterAccountAsGuardianResponse);
 
+  rpc CreateAllowlist(MsgCreateAllowlistRequest) returns (MsgAllowlistResponse);
+  rpc DeleteAllowlist(MsgDeleteAllowlistRequest) returns (MsgAllowlistResponse);
```

Comment:
nits: `CreateAllowlistEntry` and `DeleteAllowlistEntry`, since these don't create/delete entire validator allowlists